skimage/transform/_geometric.py

```diff
@@ -570,8 +570,32 @@ class ProjectiveTransform(GeometricTransform):
     params : (D+1, D+1) array
         Homogeneous transformation matrix.
 
-    """
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from skimage.transform import ProjectiveTransform
+    >>> from skimage import data
+    >>> img_src = data.rocket()
+    >>> height, width, dim = img_src.shape
+    >>> img_dst = np.zeros((height, width, dim))
```

Comment:
You could save one line / assignment here, creating your array of zeroes with: `img_dst = np.zeros(img_src.shape)`
But, wait, I can see that `height, width` are used later on... and what about `img_dst`?

Reply:
Great spot. I will work on an improved version.